crypto/mem_sec.c

```diff
@@ -31,6 +31,10 @@
 # include <unistd.h>
 # include <sys/types.h>
 # include <sys/mman.h>
+# if defined(OPENSSL_SYS_LINUX)
```

Comment:
We haven't actually come to an agreement within the team, and we probably should.  Some of us tend to prefer OPENSSL_SYS_xxx while others prefer to trust the compiler.  You will currently get a different response depending on which one of us you talk with...

This is probably part of the technical debt we're due...